crates/tx5/tests/tests/events.rs

```diff
@@ -0,0 +1,199 @@
+use std::sync::Arc;
+
+async fn sbd() -> sbd_server::SbdServer {
+    let config = sbd_server::Config {
+        bind: vec!["127.0.0.1:0".to_string(), "[::1]:0".to_string()],
+        limit_clients: 100,
+        disable_rate_limiting: true,
```

Comment:
Should we have done this on the bootstrap2 server? I thought the other day when we were talking about 0.4 vs 0.5 that we hadn't done that with the new bootstrap servers but it slipped my mind again.